### ISSUE TITLE
Show which component is used?

### ISSUE THREAD

**Issue by ElKevo** [Perfare/UnityStudio#35]
Hey, 
the scene Hierarchie view is quite nice, but would it be also somehow possible to see which components/script is used on each game object?

**Comment by Stellarspace** [comment]
Yes probably, but someone would have to implement it.

**Comment by ordigdug** [comment]
Already working on it

**Comment by ElKevo** [comment]
nice, thanks for the answer :)

**Comment by ordigdug** [comment]
No ETA, I have started the basic groundwork but a lot more has to be done.

**Comment by ElKevo** [comment]
alright, thanks for your hard work!

**Comment by GalatasarayForever** [comment]
yes that would be great to see the data connections and scripts in hierarchie

**Comment by ForgedInception** [comment]
Any update on this feature ?

**Comment by bsimser** [comment]
Yeah I'm frustrated with the scene hierarchy as it doesn't give me any info other than the contents of a scene. Going to do a spike where you can get the details from the objects in the hierarchy and cross reference them with the gameobject/asset. If you guys want it will issue a PR. Thanks.
